spine-godot/spine_godot/SpineSprite.cpp

```diff
@@ -665,15 +665,15 @@ void SpineSprite::update_skeleton(float delta) {
 		!animation_state->get_spine_object())
 		return;
 
-	emit_signal("before_animation_state_update", this);
+	emit_signal(SNAME("before_animation_state_update"), this);
```

Comment:
These `SNAME`s here are the only ones that will possibily make a difference in terms of #allocations, as they are created each frame. The rest won't hurt, but they won't make any difference performance wise.

Reply:
YA, some definitely not needed, but I just opted for consistency everywhere in spine_godot .  My convention is to always use SNAME for any `const char*`  godot API that takes `StringName`.

Reply:
Makes sense!